spec/query_spec.rb

```diff
@@ -70,6 +70,64 @@
     end
   end
 
+  describe "#start_time=" do
+    before do
+      subject.pattern    = "semla"
+      subject.start_time = time
+    end
+
+    context "when given time in UTC" do
+      let(:time) { Time.new(2016, 2, 9, 9, 01, 22, "+00:00") }
```

Comment:
Not sure this really matters, but

```
irb(main):003:0> Time.new(2016, 2, 9, 9, 01, 22, "+00:00").utc?
=> false
irb(main):004:0> Time.new(2016, 2, 9, 9, 01, 22, "+00:00").utc.utc?
=> true
```

Maybe we should use `Time.parse`? Then it's possible to use "UTC" (which seems what Ruby needs to interpret something as UTC)

```
irb(main):014:0> Time.parse("2016-02-09 09:01:22 +0000").utc?
=> false
irb(main):015:0> Time.parse("2016-02-09 09:01:22 UTC").utc?
=> true
```